src/styles/sass/bridgestatuschart.scss

```diff
@@ -0,0 +1,50 @@
+.main {
+  //border: 2px solid red;
```

Comment:
Can probably delete these commented out borders

Reply:
Yes,we can.